src/utils/renders-element.tsx

```diff
@@ -0,0 +1,4 @@
+import * as React from 'react';
```

Comment:
I'd move this to utils/types.tsx

Reply:
it's reexported there, but I need this in a separated file to be able to have a flow override